central/activecomponent/updater/updater_impl.go

```diff
@@ -25,7 +25,7 @@ var (
 
 	updaterCtx = sac.WithGlobalAccessScopeChecker(context.Background(),
```

Comment:
Probably not for now, but IMO all inits of scope contexts to fulfill internal workflows should be changed to `sac.WithGlobalAccess(context.Background())`. We know the purpose of each init `ctx` such as, building index at startup, or removing dependent resources, etc., and that these are used internally only. There is a risk, which historically has realized more than expected, that some feature begins depending on additional sac resources and insufficient context cause is not the obvious.